nservicebus/upgrades/6to7/index.md

```diff
@@ -182,6 +182,30 @@ Setting a custom [correlation ID](/nservicebus/messaging/headers.md#messaging-in
 
 In NServiceBus Version 6, the `Conversation Id` header on outgoing messages was set within the `IOutgoingPhysicalMessageContext` pipeline stage. In NServiceBus Version 7 and above, the `Conversation Id` header will be set as part of the `IOutgoingLogicalMessageContext` stage.
 
+
 ## ConfigurationErrorsException
 
-Exceptions of type `Exception` are now thrown instead of `ConfigurationErrorsException`. Any try-catch statements catching `ConfigurationErrorsException` should be updated to catch `Exception` instead.
\ No newline at end of file
+Exceptions of type `Exception` are now thrown instead of `ConfigurationErrorsException`. Any try-catch statements catching `ConfigurationErrorsException` should be updated to catch `Exception` instead.
+
+
+## Licensing
+
+### Machine wide license locations
+
+License files can now be stored on the local file system to be accessed by all endpoints running on this machine. By default, NServiceBus endpoints will check the following locations for a `license.xml` file :
+* `{Environment.SpecialFolder.LocalApplicationData}\ParticularSoftware`
+* `{Environment.SpecialFolder.CommonApplicationData}\ParticularSoftware`
+
+Note: These locations are dependent on your operating system and configuration.
```

Comment:
this would seem redundant given u have expressed to locations with `Environment.SpecialFolder....`